pinot-query-planner/src/main/java/org/apache/pinot/query/planner/serde/ProtoSerializationUtils.java

```diff
@@ -215,7 +215,7 @@ private static Object constructLiteral(Plan.LiteralField literalField) {
       case STRINGFIELD:
         return literalField.getStringField();
       case BYTESFIELD:
-        return literalField.getBytesField();
+        return literalField.getBytesField().toByteArray();
```

Comment:
in line 150 the input format is `byte[]` we need to convert that to ByteArray as well if we are doing the deser into byte array. they need to match.

Reply:
changed.